coderd/database/queries/insights.sql

```diff
@@ -661,7 +661,7 @@ WITH
 			AND date_trunc('minute', was.created_at) = mb.minute_bucket
 			AND was.template_id = mb.template_id
 			AND was.user_id = mb.user_id
-			AND was.connection_median_latency_ms >= 0
```

Comment:
This is the bug fix, in other queries that use `connection_median_latency_ms` we always filter our the 0 values to rule out any usage stats.